packages/suite-desktop/src/modules/bridge.ts

```diff
@@ -20,6 +34,37 @@ const load = async () => {
         bridge.stop();
     });
 
+    ipcMain.handle('bridge/toggle', async (_: unknown) => {
+        const status = await handleBridgeStatus(bridge, mainWindow);
```

Comment:
In a single `bridge/toggle`, `bridge/status` will be emitted twice. Is it expected and/or required?

Reply:
not required but it does not hurt anybody. also not nice. again, open to discuss it in the office.